cpp/src/groupby/sort/group_single_pass_reduction_util.cuh

```diff
@@ -38,40 +43,30 @@ namespace groupby {
 namespace detail {
 
 /**
- * @brief ArgMin binary operator with index values into input column.
+ * @brief Binary operator with index values into the input column.
  *
  * @tparam T Type of the underlying column. Must support '<' operator.
  */
-template <typename T>
-struct ArgMin {
+template <typename T, bool has_nulls, bool arg_min>
```

Comment:
Does `has_nulls` need to be a template parameter here? That is, could it be stored as a member variable instead?
This could potentially save alot of generated code that is only different by this part of 2 if-clauses: `has_nulls && d_col.is_null_nocheck(lhs/rhs_idx)`

Reply:
Yes, it is possible. Thanks for this.